clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybookURITest.spec.ts

```diff
@@ -15,8 +15,10 @@ import * as path from "path";
 import * as vscode from "vscode";
 import { CopybookURI } from "../../../services/copybook/CopybookURI";
 import { Utils } from "../../../services/util/Utils";
+import { ZOWE_FOLDER } from "../../../constants";
 
 Utils.getZoweExplorerAPI = jest.fn();
+Utils.getExtensionFolder = jest.fn().mockReturnValue(__dirname);
```

Comment:
This would not be needed if `createCopybookPath` took an extra argument.

Reply:
Its about refactoring the current. These will gonna change anyway in e4e copybook PR. I'd say let's not duplicate the work.

Reply:
It is indeed about old code.  workspace path wasn't passed as argument into `createCopybookPath`. So I kept it same. refactored at : 652cb705445212aca0650c9faf4fa53e2b0c4a48